docs/getting-started/bpfctl-guide.md

```diff
@@ -229,19 +237,211 @@ program's return value. For example, the default `proceed-on` configuration for
+ 6201       96c4671c-e764-4016-8e79-ee99b2d58c12  pass              xdp            2023-07-17T17:17:53-0400
+ 6202       995e87fe-4d1d-48ce-b348-3411342cf661  sys_enter_openat  tracepoint     2023-07-17T17:19:09-0400
+ 6203                                             dispatcher        tc             2023-07-17T17:20:14-0400
+ 6204       665954eb-0532-4849-8db6-e127e3fe3072  stats             tc             2023-07-17T17:20:14-0400
+ 6207                                             xdp               xdp            2023-07-17T17:27:13-0400
+```
+
+To filter on a given program type, include the `--program-type` parameter:
+
+```console
+bpfctl list --all --program-type tc
+ Kernel ID  Bpfd UUID                             Name        Type  Load Time
+ 6203                                             dispatcher  tc    2023-07-17T17:20:14-0400
+ 6204       665954eb-0532-4849-8db6-e127e3fe3072  stats       tc    2023-07-17T17:20:14-0400
+```
+
+## bpfctl get
+
+To retrieve detailed information for a loaded eBPF program, use the
+`bpfctl get` command.
+If the eBPF program was loaded via bpfd, then there will be a `Bpfd State`
+section with bpfd related attributes and a `Kernel State` section with
+kernel information.
+If the eBPF program was loaded outside of bpfd, then the `Bpfd State`
+section will be empty and `Kernel State` section will be populated.
+
+```console
+bpfctl get 6204
+
+#################### Bpfd State ####################
+
+UUID:                               665954eb-0532-4849-8db6-e127e3fe3072
+Image URL:                          quay.io/bpfd-bytecode/go-tc-counter:latest
+Pull Policy:                        IfNotPresent
+Global:                             None
+Map Pin Path:                       /run/bpfd/fs/maps/665954eb-0532-4849-8db6-e127e3fe3072
+Map Owner UUID:                     None
+Map Used By:                        None
+Priority:                           100
+Iface:                              vethff657c7
+Position:                           0
+Direction:                          eg
+Proceed On:                         pipe, dispatcher_return
+
+#################### Kernel State ##################
+
+Kernel ID:                          6204
+Name:                               stats
+Type:                               tc
+Loaded At:                          2023-07-17T17:20:14-0400
+Tag:                                ead94553702a3742
+GPL Compatible:                     true
+Map IDs:                            [2705]
+BTF ID:                             2821
+Size Translated (bytes):            176
+JITed:                              true
+Size JITed (bytes):                 116
+Kernel Allocated Memory (bytes):    4096
+Verified Instruction Count:         24
+```
+
+```console
+bpfctl get 6190
+
+#################### Bpfd State ####################
+NONE
+
+#################### Kernel State ##################
+
+Kernel ID:                          6190
+Name:
```

Comment:
Hrm this is probably a field we should put `None` for if the name is empty.

Reply:
Maybe a small fix in another issue?

Reply:
Done